extensions/spring-xd-extension-cassandra/src/main/java/org/springframework/xd/cassandra/mixins/CassandraConnectionMixin.java

```diff
@@ -44,7 +44,7 @@
 
 	private String initScript;
 
-	private String[] entityBasePackages = new String[0];
+	private String[] entityBasePackages;
```

Comment:
@ilayaperumalg, have you tested it before pushing?
The reason for the `new String[0]` is here (`AbstractCassandraConfiguration`):
```
/**
 * The base packages to scan for entities annotated with {@link Table} annotations. By default, returns the package
 * name of {@literal this} (<code>this.getClass().getPackage().getName()</code>). This method must never return null.
 */
public String[] getEntityBasePackages() {
	return new String[] { getClass().getPackage().getName() };
}
```
And so on up to the:
```
  ArrayList(E[] array) {
         a = Objects.requireNonNull(array);
}

```